src/test/java/org/opensearch/security/dlic/rest/api/AuditApiActionTest.java

```diff
@@ -564,15 +581,16 @@ private String getTestPayload() {
         return "{" +
                 "\"enabled\":true," +
                 "\"audit\":{" +
-                    "\"enable_rest\":true,\"disabled_rest_categories\":[\"AUTHENTICATED\"]," +
```

Comment:
Please take this as an action item for follow up PR. The indentation follows json formatting and was correct before changes.